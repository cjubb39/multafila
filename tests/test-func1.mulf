/* function declaration & function calls */


int mul_helper(int x, int y){
	int result = x * y;
	return result;

}
int main(){

	int i = 4;
	int j = 7;
	
	int mul = mul_helper(i, j);
	
	printOut(mul); /*should print 28 */
	
	return 1;
	
}
