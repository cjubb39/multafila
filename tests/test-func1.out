
/*==========OUTPUT CODE BELOW==========*/
#include <stdio.h>
#include <pthread.h>
void printOut(char *m){printf("%s\n", m);}
void printInt(int m){printf("%d\n", m);}

int mul_helper (  int x,  int y );
int main (  );

pthread_mutex_t global_lock;

pthread_cond_t global_cond;

struct global_lock_struct_s{
} global_lock_struct;

pthread_t global_threads [0];
int mul_helper (  int x,  int y )
{
result = x * y ;
return result ;
}

int main (  )
{

{
pthread_mutex_init( & global_lock , NULL);
pthread_cond_init( & global_cond , NULL);
}
 ;
i = 4 ;
j = 7 ;
mul = mul_helper( i, j ) ;
printOut( mul ) ;
return 1 ;
}

